spec/features/allocations_spec.rb

```diff
@@ -5,13 +5,16 @@
 
   scenario "Accredited provider views PE allocations page" do
     given_accredited_body_exists
+    given_training_provider_with_pe_fee_founded_course_exists
```

Comment:
I’m aware that this methods doesn’t really do what it says. Since the filtering is done on the api side and we are stubbing the response on line 71 `resource_list_to_jsonapi([@training_provider1])`. I can’t see the point of creating subjects, courses, etc, which would slow down the tests but I think it helps to document the intent.